<launch>

<rosparam command="load" file="home_directory.yaml"/>
<rosparam command="load" file="csv_data_filename.yaml"/>

<rosparam command="load" file="camera_parameters_basler.yaml"/>
<node name="camera" pkg="camera_aravis" type="camnode" />

<rosparam command="load" file="tracker_parameters.yaml"/>
<rosparam command="load" file="data_association_parameters.yaml"/>

<node name="tracker" pkg="multi_tracker" type="tracker_simplebuffer.py" output="screen" />
<node name="data_association" pkg="multi_tracker" type="data_association.py" output="screen" />
<node name="delta_video" pkg="multi_tracker" type="delta_video_simplebuffer.py" output="screen" />
<node name="save_hdf5_data" pkg="multi_tracker" type="save_data_to_hdf5.py" output="screen" />

<node name="save_delta_video" pkg="multi_tracker" type="save_bag.py" args="--config=/home/caveman/orchard/src/delta_video_config.py" output="screen" />
<node name="save_raw_data" pkg="multi_tracker" type="save_bag.py" args="--config=/home/caveman/orchard/src/raw_data_bag_config.py" output="screen" />



</launch>
